README.md

```diff
@@ -10,24 +10,13 @@
   <p>Import and process HTML Modules with PostHTML</p>
 </div>
 
-<h2 align="center">Install</h2>
+## Install
 
 ```bash
-npm i -D posthtml-modules
+npm i posthtml-modules
```

Comment:
can we add the `-D` as it should be installed as dev dep right !

Reply:
Sure thing! Added back, sorry about that!